Extensions/video_downloader.js

```diff
@@ -0,0 +1,52 @@
+//* TITLE Video Downloader **//
+//* VERSION 1.0.0 **//
+//* DESCRIPTION	adds a download button to the video player **//
+//* DETAILS Adds a simple download button to all videos using the fancy player. Does not support 3rd-party players or the native player. Disabled on beta dash (now supported natively)**//
+//* DEVELOPER tlitookilakin **//
+//* FRAME false **//
+//* BETA false **//
+XKit.extensions.video_downloader = new Object({
+
+	running: false,
+
+	run: function() {
+		this.running = true;
+		if (!XKit.interface.is_tumblr_page() || XKit.page.react) {
+			return;
+		}
+		XKit.tools.init_css("video_downloader");
+		XKit.extensions.video_downloader.addButtons();
+		XKit.post_listener.add("video_downloader", XKit.extensions.video_downloader.addButtons);
```

Comment:
you can use `this` in place of `XKit.extensions.video_downloader` in both places here

Reply:
Considering these lines were _directly_ copied from the example in the documentation, I would suggest updating it to conform to submission standards. Also, you have write access to the repository now, so if you want to make any edits, go wild.

Reply:
it was just a suggestion, sorry. what's here is fine! i'm just used to reviewing things maybe a little too thoroughly...